src/Core/src/Handlers/ShapeView/ShapeViewHandler.Windows.cs

```diff
@@ -1,11 +1,11 @@
-﻿using Microsoft.Maui.Graphics.Win2D;
+﻿using Microsoft.Maui.Graphics.Platform;
 
 namespace Microsoft.Maui.Handlers
 {
-	public partial class ShapeViewHandler : ViewHandler<IShapeView, W2DGraphicsView>
+	public partial class ShapeViewHandler : ViewHandler<IShapeView, MauiShapeView>
```

Comment:
This is a breaking change in CORE. Instead of all the platforms using `MauiShapeView` and windows being a bit "special", we all now use this same type: `MauiShapeView`. It is very simple and does not do anything special:

```cs
public class MauiShapeView : PlatformGraphicsView
{
}
```